css/about.css

```diff
@@ -0,0 +1,104 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
In the future, if you want you can move the normalize/reset CSS into another file as well so you don't have to copy/paste.  Then just reference it before you reference the about/index.css files